package.json

```diff
@@ -49,17 +49,17 @@
     "eslint-plugin-promise": "^5.1.0",
     "ethereum-waffle": "^3.4.0",
     "ethereumjs-util": "^7.1.4",
-    "ethers": "^5.5.4",
-    "hardhat": "^2.9.1",
+    "ethers": "^5.7.1",
+    "hardhat": "^2.12.0",
     "hardhat-deploy": "^0.10.5",
     "husky": "^4.2.3",
     "prettier": "^2.3.2",
-    "prettier-plugin-solidity": "^1.0.0-beta.13",
+    "prettier-plugin-solidity": "^1.0.0-beta.24",
     "pretty-quick": "^2.0.1",
     "solidity-coverage": "^0.7.16",
     "ts-node": "^10.1.0",
-    "typechain": "^5.1.2",
-    "typescript": "^4.5.2"
+    "typechain": "^8.1.0",
```

Comment:
I have updated some packages, I thought it would be good to have them up to date. All tests are green.